content/libraries/klc_history.adoc

```diff
@@ -5,6 +5,12 @@ url = "/libraries/klc/history/"
 +++
 
 ---
+== 3.0.22 - 2019-09-20
+* Clarified KiCad footprint assignment work-flows as generic vs fully specified symbols (G2.1)
```

Comment:
Is this hyphen required? Do you want to hyphenate `fully-specified` or leave it as two separate words?

Reply:
no